packages/core-util/src/sdks/colorPalette.ts

```diff
@@ -37,6 +36,7 @@ export const colorPalette = [
   '#5f6b7a',
   '#3759ce',
   '#b1325c',
+  '#5978e3',
```

Comment:
What's the purpose of moving this color?

Reply:
That blue is very similar to the blue at index 2. Moved it further down and made sure that 4 colors above and below are not similar.